babelOptions.js

```diff
@@ -0,0 +1,9 @@
+exports.getBabelOptions = ({useESModules}) => ({
```

Comment:
Do you remember why we didn't use babel.config.js?

Reply:
no, but I would guess the useESModules option is dynamic, that's why?

Reply:
Yes, but it can be used only for rollup. There is no difference between inlined and imported helpers.

Reply:
using the option for rollup only is fine, it's still better than having the entire babel config duplicted

Reply:
Btw, babel 7.13 does not require useESModules flag anymore. Right version is detected automatically.

Reply:
then we can remove the flag along with upgrading babel, probably better separately